protostar/commands/invoke_command.py

```diff
@@ -124,8 +124,7 @@ async def invoke(
             contract_address=contract_address,
             function_name=function_name,
             inputs=inputs,
-            max_fee=max_fee if isinstance(max_fee, int) else None,
-            auto_estimate_fee=max_fee == "auto",
+            max_fee=max_fee if max_fee is not None else "auto",
```

Comment:
```suggestion
            max_fee=max_fee
```
I think `max_fee` shouldn't be `None` at this point.